eng/tools/generator/cmd/v2/common/generation.go

```diff
@@ -213,7 +213,7 @@ func (ctx *GenerateContext) GenerateForSingleRPNamespace(generateParam *Generate
 	if !onBoard {
 		log.Printf("Get ori exports for changelog generation...")
 
-		tags, err := GetAllVersionTags(generateParam.RPName, generateParam.NamespaceName)
+		tags, err := GetAllVersionTags("sdk/resourcemanager/" + generateParam.RPName + "/" + generateParam.NamespaceName)
```

Comment:
please consolidate to using `filepath.Join`.

Reply:
used fmt.Sprintf()